packages/terra-theme-provider/src/ThemeProvider.jsx

```diff
@@ -18,52 +18,42 @@ const propTypes = {
    * Name of class for specified theme. e.g `cerner-consumer-theme`
    */
   themeName: PropTypes.string,
-  /**
-   * When set to true, applies theme class to HTML element
-   */
-  isGlobalTheme: PropTypes.bool,
-};
-
-const defaultProps = {
-  isGlobalTheme: false,
 };
 
 class ThemeProvider extends React.Component {
   componentDidMount() {
-    if (this.props.isGlobalTheme === true && this.props.themeName) {
+    if (this.props.themeName) {
       document.documentElement.classList.add(this.props.themeName);
     }
   }
 
   componentDidUpdate(prevProps) {
     if (this.props === prevProps) return;
-    if (this.props.isGlobalTheme === true) {
-      if (prevProps.themeName) {
-        document.documentElement.classList.remove(prevProps.themeName);
-      }
 
-      if (this.props.themeName) {
-        document.documentElement.classList.add(this.props.themeName);
-      }
+    if (prevProps.themeName) {
+      document.documentElement.classList.remove(prevProps.themeName);
+    }
+
+    if (this.props.themeName) {
+      document.documentElement.classList.add(this.props.themeName);
     }
   }
 
   componentWillUnmount() {
-    if (this.props.isGlobalTheme === true && this.props.themeName) {
+    if (this.props.themeName) {
       document.documentElement.classList.remove(this.props.themeName);
     }
   }
 
   render() {
     const {
-      themeName, isGlobalTheme, children, ...customProps
+      themeName, children, ...customProps
     } = this.props;
     return (<div {...customProps} className={cx(themeName, customProps.className)}>{children}</div>);
```

Comment:
Theoretically, you could remove the themeName from this div, but you can't remove the div.

Reply:
Yes, I verified after removing themeName from here and it works as expected.
So, addressed it here : https://github.com/cerner/terra-framework/pull/1031/commits/0c05483821789068554300b3945aaf7198b959be